package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/main.js",
   "scripts": {
     "build": "rollup --config",
-    "start": "webpack-dev-server --mode development",
```

Comment:
this code was launching this error
![image](https://user-images.githubusercontent.com/29656850/115025233-7c719180-9e97-11eb-82a6-7b61f5621925.png)